src/controllers/DynamicFacetQueryController.ts

```diff
@@ -6,11 +6,13 @@ import { QueryEvents } from '../events/QueryEvents';
 import { findIndex } from 'underscore';
 import { IQueryResults } from '../rest/QueryResults';
 import { IDynamicFacet } from '../ui/DynamicFacet/IDynamicFacet';
+import { Utils } from '../Core';
 
 export class DynamicFacetQueryController {
   private numberOfValuesToRequest: number;
   private freezeCurrentValues = false;
   private freezeFacetOrder = false;
+  private hasFolding = false;
```

Comment:
From "quickly" looking at the code, it seems we could somehow make due without putting this as a property on the class and instead keep a local variable ? Always better if we don't have to maintain a state variable/flag like this.

Reply:
A local variable would require to pass the querybuilder has a parameter to facetRequest